ReactiveCocoa/Swift/Property.swift

```diff
@@ -430,33 +456,18 @@ public struct AnyProperty<Value>: PropertyType {
 		}
 	}
 
-	/// Check if `property` is an `AnyProperty` and has already captured its sources
+	/// Check if `property` is an `Property` and has already captured its sources
 	/// using a closure. Returns that closure if it does. Otherwise, returns a closure
 	/// which captures `property`.
 	private static func capture<P: PropertyType>(property: P) -> [Any] {
-		if let property = property as? AnyProperty<P.Value> {
+		if let property = property as? Property<P.Value> {
```

Comment:
@mdiep See the rationale, or you may check the PR for this. This is crucial to the semantics you suggested. 🐵